assets/cubyz/biomes/island.zig.zon

```diff
@@ -16,16 +16,10 @@
 	},
 	.structures = .{
 		.{
-			.id = "cubyz:simple_tree",
-			.leaves = "cubyz:oak_leaves",
-			.log = "cubyz:oak_log",
-			.top = "cubyz:oak_top",
-			.chance = 0.016,
-			.type = .round,
-			.height = 12,
-			.height_variation = 10,
-			.leafRadius = 5,
-			.leafRadius_variation = 6,
```

Comment:
islands, being a rather rare thing, should not have ordinary trees in my opinion.